androidHyperskillApp/src/main/java/org/hyperskill/app/android/first_problem_onboarding/fragment/FirstProblemOnboardingFragment.kt

```diff
@@ -0,0 +1,74 @@
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.ViewModelProvider
+import org.hyperskill.app.android.HyperskillApp
+import org.hyperskill.app.android.core.view.ui.navigation.requireAppRouter
+import org.hyperskill.app.android.core.view.ui.widget.compose.HyperskillTheme
+import org.hyperskill.app.android.first_problem_onboarding.ui.FirstProblemOnboardingScreen
+import org.hyperskill.app.core.view.handleActions
+import org.hyperskill.app.first_problem_onboarding.presentation.FirstProblemOnboardingFeature.Action.ViewAction
+import org.hyperskill.app.first_problem_onboarding.presentation.FirstProblemOnboardingViewModel
+import ru.nobird.android.view.base.ui.extension.argument
+import ru.nobird.android.view.base.ui.extension.snackbar
+
+class FirstProblemOnboardingFragment : Fragment() {
+    companion object {
+        const val FIRST_PROBLEM_ONBOARDING_FINISHED = "FIRST_PROBLEM_ONBOARDING_FINISHED"
+        fun newInstance(isNewUserMode: Boolean): FirstProblemOnboardingFragment =
+            FirstProblemOnboardingFragment().apply {
+                this.isNewUserMode = isNewUserMode
+            }
+    }
+
+    private var isNewUserMode: Boolean by argument()
+
+    private var viewModelFactory: ViewModelProvider.Factory? = null
+    private val firstProblemOnboardingViewModel: FirstProblemOnboardingViewModel by viewModels {
+        requireNotNull(viewModelFactory)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        injectComponent()
+        firstProblemOnboardingViewModel.handleActions(this, block = ::onAction)
+    }
+
+    private fun injectComponent() {
+        val platformFirstProblemOnboardingComponent =
+            HyperskillApp.graph().buildPlatformFirstProblemOnboardingComponent(isNewUserMode)
+        viewModelFactory = platformFirstProblemOnboardingComponent.reduxViewModelFactory
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View =
+        ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnLifecycleDestroyed(viewLifecycleOwner))
+            setContent {
+                HyperskillTheme {
+                    FirstProblemOnboardingScreen(viewModel = firstProblemOnboardingViewModel)
+                }
+            }
+        }
+
+    private fun onAction(action: ViewAction) {
+        when (action) {
+            is ViewAction.CompleteFirstProblemOnboarding -> {
+                requireAppRouter().sendResult(FIRST_PROBLEM_ONBOARDING_FINISHED, action.stepRoute ?: Any())
+            }
+            ViewAction.ShowNetworkError -> {
+                requireView().snackbar(org.hyperskill.app.R.string.common_error)
```

Comment:
```suggestion
                requireView().snackbar(org.hyperskill.app.R.string.first_problem_onboarding_network_error)
```

Reply:
Sorry, this resource is currently available only in the iOS PR
https://github.com/hyperskill/mobile-app/pull/702/files#diff-37756673d1942b34ce2f8464c22ec4f77215a8d81bc0a274194351759f6cc369

Reply:
Ok, I'll wait for it, and then merge